R/tbl_likert.R

```diff
@@ -130,37 +140,198 @@ tbl_likert <- function(data,
   result
 }
 
-
+#' Add column with N to a Likert table
 #' @export
-#' @rdname tbl_likert
-add_n.tbl_likert <- function(x, ...) {
+#' @inheritParams gtsummary::add_n.tbl_summary
+#' @param x Object with class `tbl_likert` from the [tbl_likert()] function
+add_n.tbl_likert <- function(x,
+                             statistic = "{n}",
+                             col_label = "**N**",
+                             footnote = FALSE,
+                             last = FALSE,
+                             ...) {
   # check inputs ---------------------------------------------------------------
   if (!inherits(x, "tbl_likert"))
     stop("`x=` must be class 'tbl_likert'", call. = FALSE)
 
-  # add n column ---------------------------------------------------------------
+  data <- x$inputs$data
+  include <- x$inputs$include
+
+  if ("n" %in% names(x$table_body))
+    stop("`add_n()` has already been applied", call. = FALSE)
+
+  # compute stat ---------------------------------------------------------------
+  data <-
```

Comment:
In `tbl_likert()` we used `fct_unify()` to unify all the levels of the variables, but here that is not being taken into account.

Reply:
Would it be easier and more consistent to save the unified version of `data` as proposed here: https://github.com/ddsjoberg/bstfun/pull/62/commits/048045cd6afbe82e78552427b913169da9a66f57 ?

Reply:
For these functions, it is not an issue, as we are just adding additional stat col but not modifying the labelling